.github/workflows/openmldb-docker.yml

```diff
@@ -66,6 +66,7 @@ jobs:
 
       - name: Build And Push
         id: docker_build
+        if: ${{ github.event_name == 'workflow_dispatch' }}
```

Comment:
no need .
this step also include build, push is controlled by `${env.PUSH}`

Reply:
It won't push but it will test, and the arg OPENMLDB_VERSION will be empty

Reply:
u mean docker build failed in pr or push context, looks like a bug then

Reply:
`openmldb==$OPENMLDB_VERSION` is the change in this pr. Original pr docker test will use run `/openmldb.sh ""` in stage `Build And Push`, so it'll use the default version in `setup_openmldb.sh`.
![image](https://user-images.githubusercontent.com/24697960/209931621-dc4de626-d028-4a4e-8556-ebfe906b8ba3.png)
![image](https://user-images.githubusercontent.com/24697960/209931717-438cef56-c958-4a8a-bdbf-20eeec7ec444.png)